src/nvim/api/vim.c

```diff
@@ -575,6 +575,25 @@ Object nvim_notify(String msg, Integer log_level, Dictionary opts, Error *err)
   return nlua_exec(STATIC_CSTR_AS_STRING("return vim.notify(...)"), args, err);
 }
 
+/// Notify the user with a message
+///
+/// Relays the call to vim.notify . By default forwards your message in the
+/// echo area but can be overridden to trigger desktop notifications.
```

Comment:
Hmm the doc is wrong here, it's not a notification system but a picker system, a way to select one thing in a list

Reply:
thanks for reminding me I am a messy committer :p